manifests/art.yaml

```diff
@@ -0,0 +1,16 @@
+updates:
+  - file: "{MAJOR}.{MINOR}/cluster-logging.v{MAJOR}.{MINOR}.0.clusterserviceversion.yaml" # relative to this file
+    update_list:
+    # replace metadata.name value
+    - search: "clusterlogging.v{MAJOR}.{MINOR}.2"
```

Comment:
the reason we are replacing a ".2" is because we have a `4.1.2` csv? i don't see that version in master to be replaced...

Reply:
i think you're right, this needs to be `clusterlogging.v{MAJOR}.{MINOR}.0` i think, to match https://github.com/openshift/cluster-logging-operator/pull/204/files#diff-5766fe294b4d426421892b30ee5365eaR257

will fix.